.github/workflows/main.yml

```diff
@@ -2,20 +2,31 @@
 name: CI
 
 on:
+  # using pull_request ensures CI can run from PRs from forks
```

Comment:
remind me - PRs from forks will run without our secrets, right?

Reply:
My understanding is that `pull_request` triggers the workflow with

a) read only GITHUB_TOKEN
b) no secrets

This could be an issue if we use secrets in the actual tests (e.g. job-server currently requires a RO github PAT for tests). We should probably:

a) not do that if possible
b) skip those tests rather than fail them

More here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflows-in-forked-repositories